pages/category/[slug].tsx

```diff
@@ -0,0 +1,44 @@
+import type { FunctionComponent } from 'react';
+import type { Story } from '@prezly/sdk';
+import { GetServerSideProps } from 'next';
+import { getPrezlyApi } from '@/utils/prezly';
+import Layout from '@/components/Layout';
+import Stories from '@/modules/Stories';
+import { Category } from '@prezly/sdk/dist/types';
+
+type Props = {
+    stories: Story[];
+    category: Category
+    categories: Category[]
+};
+
+const IndexPage: FunctionComponent<Props> = ({ category, stories, categories }) => (
+    <Layout categories={categories}>
+        <h1>{category.display_name}</h1>
+        <p>
+            {category.display_description}
+        </p>
+        <Stories stories={stories} />
+    </Layout>
+);
+
+export const getServerSideProps: GetServerSideProps<Props> = async (context) => {
+    const api = getPrezlyApi(context.req);
+    const { slug } = context.params as { slug: string };
+    const categories = await api.getCategories();
+    const category = await api.getCategoryBySlug(slug);
+
+    if (!category) {
+        return {
```

Comment:
Inline vs multiline object: https://github.com/prezly/theme-nextjs-starter/pull/7/files#diff-f3bbbdb92fd1d1da397061576d4275aaadbcdf1e1952d4cbe94b5928478141fcR24

Probably should be also established to have in consistent everywhere.